### PR TITLE
Task2a - Populated DB

### PR DESCRIPTION
**Pull request by Sen-i**

### CODE REVIEW

File: api/populateDatabase.php [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+//}
+
+//foreach ($statusResult as $status) {
+//    insertIntoStatusTable($database, $status);
+//}
+
+//foreach ($typeResult as $type) {
+//    insertIntoTypesTable($database, $type);
+//}
+
```
Comment (bradleypurdue) [inline api/populateDatabase.php]: Don't commit commented code.

File: api/populateDatabase.php [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+                                                                :ADDRESS_1, 
+                                                                :ADDRESS_2, 
+                                                                :TOWN, 
+                                                                :POSTCODE, 
+                                                                :DESCRIPTION, 
+                                                                :BEDROOMS, 
+                                                                :PRICE, 
+                                                                :IMAGE, 
+                                                                :TYPE, 
+                                                                :STATUS);");
```
Comment (bradleypurdue) [inline api/populateDatabase.php]: This is really annoying to read. Did PHPStorm recommend you to do this?

File: api/populateDatabase.php [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    return $query->execute($propertyData);
+}
+
+function insertIntoStatusTable(PDO $db, array $statusData): bool
+{
+    $query = $db->prepare("INSERT INTO `status` (`StatusId`,
+                                                            `StatusName`) 
+                                                        VALUES (:ID, 
+                                                                :STATUS_NAME);");
```
Comment (bradleypurdue) [inline api/populateDatabase.php]: Same with this.

File: api/populateDatabase.php [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    return $query->execute($statusData);
+}
+
+function insertIntoTypesTable(PDO $db, array $typeData): bool
+{
+    $query = $db->prepare("INSERT INTO `types` (`TypeId`,
+                                                            `TypeName`) 
+                                                        VALUES (:ID, 
+                                                                :TYPE_NAME);");
```
Comment (bradleypurdue) [inline api/populateDatabase.php]: And this.

File: api/populateDatabase.php [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+    return $query->execute($typeData);
+}
+
+
+
+function validateUsrInput(array $newBread) :bool
```
Comment (bradleypurdue) [inline api/populateDatabase.php]: Shouldn't have a space before `:`, but after it instead.

**Review by turnerror — CHANGES_REQUESTED** [review]
There are a lot allow nulls in the table structure that probably shouldn't be allow null.
ALSO the fieldnames should be lower case because then we don't need to have properties that start with a capital letter in our objects.

File: api/populateDatabase.php (hunk 4 above)
Comment (Sen-i) [reply api/populateDatabase.php]: edited this!

File: api/populateDatabase.php (hunk 5 above)
Comment (Sen-i) [reply api/populateDatabase.php]: Removed

File: api/populateDatabase.php (hunk 3 above)
Comment (Sen-i) [reply api/populateDatabase.php]: Edited!

File: api/populateDatabase.php (hunk 2 above)
Comment (Sen-i) [reply api/populateDatabase.php]: meh it's an individuals choice and mike didn't mind so... it stays sorry :/

File: api/populateDatabase.php (hunk 1 above)
Comment (Sen-i) [reply api/populateDatabase.php]: found a better solution and updated! thanks!

**Pull request merged**